src/page/shipping-step.page.ts

```diff
@@ -5,8 +5,8 @@ export class ShippingPage {
   private checkBox: ElementFinder;
 
   constructor () {
-    this.nextButton = $('#form > p > button > span');
-    this.checkBox =  $('#cgv');
+    this.nextButton = $('.cart_navigation button[name="processCarrier"]');
+    this.checkBox =  $('.order_carrier_content input#cgv');
```

Comment:
Why is this a better solution?

Reply:
I mean, why do you put .order_carrier_content input before #cgv when #cgv was already an unique locator?